tests/installation/partitioning.pm

```diff
@@ -26,10 +26,16 @@ sub run {
         # Define changed shortcuts
         $cmd{donotformat} = 'alt-t';
         $cmd{addraid}     = 'alt-d';
-        $cmd{filesystem}  = 'alt-a'
+        $cmd{filesystem}  = 'alt-a';
 
     }
 
+    if (check_var('DISTRI', 'opensuse') && check_var('STORAGE_NG', '1')) {
+        $cmd{expertpartitioner} = 'alt-x';
+        $cmd{rescandevices}     = 'alt-c';
+        $cmd{filesystem}        = 'alt-f';
+
+    }
```

Comment:
that part should be within the upper `if (match_has_tag('storage-ng'))`. Rest is fine

Reply:
These are different between SLE 15 and Leap 15, this is why I added the second 'if'